stimuli_structure/stimuli/stimuli.tsv

```diff
@@ -0,0 +1,3 @@
+stim_id	type	NSD_id	COCO_id	first_COCO_description	HED
+nsd02951.png	still_image	2951	262145	"an open market full of people and piles of vegetables."	"((Item-count, High), Ingestible-object)), (Background-view, ((Human, Body, Agent-trait/Adult), Outdoors, Furnishing, Natural-feature/Sky, Urban, Man-made-object))"
+the_present_movie.txt	video	n/a	n/a	n/a	n/a
```

Comment:
shouldn't this be a video filetype, not .txt?

Reply:
Very good question.
Here, the movie is under copyright, so, it can't be shared freely with the BIDS dataset. In this case, the text file includes a link to publicly accesbile location for the movie.

Reply:
Resolved in the spec, now the stimulus files can be only a json sidecar, especailly when the file is not accesbile like under copyeight.